lib/mix/task.ex

```diff
@@ -20,4 +24,18 @@ defmodule Antikythera.Mix.Task do
     {:ok, _} = Application.ensure_all_started(Antikythera.Env.antikythera_instance_name())
     :ok
   end
+
+  @doc """
+  Set the specified `node_id` to the `Antikythera.ContextId` in GearLog.
+
+  If you want to use GearLog in mix task, you must set `node_id` before you call GearLog functions.
+
+  The `Antikythera.ContextId` in GearLog will become `{timestamp}_{node_id}_{PID}`.
+  The `timestamp` and `PID` are automatically got from system.
+  """
+  defun set_node_id_to_gear_log_context(node_id :: v[String.t]) :: :ok do
+    context_id = Context.make_context_id(Time.now(), node_id)
+    true = ContextId.valid?(context_id)
```

Comment:
I think it is better to define `make_context_id/2` function whose postcondition satisfies `ContextId.t`.
(To avoid checking the function output every time)

```elixir
defun make_context_id(t :: v[Time.t], node_id :: v[String.t]) :: v[ContextId.t] do
  context_id = [
    timestamp(t),
    node_id,
    :erlang.pid_to_list(self()) |> tl() |> List.to_string() |> String.trim_trailing(">"),
  ] |> Enum.join("_")
  true = ContextId.valid?(context_id)
  context_id
end
```

Reply:
It looks line `v[ContextId.t]` is enough to validate postcondition.
Fixed at https://github.com/access-company/antikythera/pull/126/commits/ce7ba1bcf65aa47d4d96f85b5b448742393616ed